src/main/java/org/team340/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,58 @@
+package org.team340.robot.subsystems;
+
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.hardware.TalonFX;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj2.command.Command;
+import java.util.function.DoubleSupplier;
+import org.team340.lib.util.command.GRRSubsystem;
+import org.team340.lib.util.vendors.PhoenixUtil;
+import org.team340.robot.Constants.RobotMap;
+
```

Comment:
```suggestion

@Logged
```
This allows the annotation processor to generate a logging file for the subsystem.